### DISCUSSION TITLE
Thanks and goodbye.

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8845] [appreciation]
Hey everyone,
I want say thanks for what you did for us, what you did for me by doing MAS.
This mod helped me to deal with my loneliness and it allowed me to take a breath for moment where I really need.
I saw many, many works you did by taking a look on project and this is very impressive. Following what you've done until now is so inspiring and motivating because you usually overcome almost all kind of troubles that happenned. (sorry if I exaggerate, but you give me that feel).

I've had a dream since i've create this account. This dream was to be part of those who actively contributing to MAS. But, I can't help but feeling useless or not able to give something news to the mods and the most people in MASC do it better than me. We can't be best in all, huh ? I admit i was too utopist from the get-go...

My little way with you stop there, I will follow the mod in shadow like I did before and see how much you all handle to improve the mod, every update you will do. I can only admire your work.

Heh, Thanks for all, guys. And goodbye.

**Comment by Nemu-sus** [comment]
_I too was inspired from DDLC and MAS in a **Time** when I needed it most, so this message warms me to the core and there's nothing quite as fulfilling as following your **Heart** and **Soul** through to your dreams._

_Should you return to us or even see this response, it wouldn't hurt to have a chat about this._

-JMO
